src/gfx/mesh_utils.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef DC6B12F5_63B6_4FBD_BC92_DFF99E0E3323
```

Comment:
Where do these names come from?

Reply:
I'm using this extension https://marketplace.visualstudio.com/items?itemName=akiramiyakoda.cppincludeguard